electron/app/services/cloud/client-config.ts

```diff
@@ -0,0 +1,16 @@
+import {type, release} from 'os';
+import {machineIdSync} from 'node-machine-id';
+import {app} from 'electron';
+
+const CLIENT_NAME = 'Monokle Desktop';
+
+export function getClientConfig() {
+	return {
+		name: CLIENT_NAME,
+		version: app.getVersion(),
+		os: `${type()} ${release()}`,
+		additionalData: {
+			machineId: machineIdSync(),
```

Comment:
Maybe for privacy purposes I'd only pass machine id when analytics are enabled?

Reply:
Fixed in https://github.com/kubeshop/monokle/pull/4206/commits/44c35df10de3884e865ec2d6b1ab568f0694ec1f.